README.md

```diff
@@ -1 +1,68 @@
-# apprentice-react-o-shop
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
simplify this read me a bit so that it doesn't have all this generated text. I would include a short description and instructions on how to run your server